cmd/common/image/helpers/Makefile.build

```diff
@@ -12,7 +12,10 @@ ARCHS = amd64 arm64
 TARGETS = $(foreach ARCH,$(ARCHS),$(OUTPUTDIR)/$(ARCH).bpf.o)
 
 .PHONY: all
-all: $(TARGETS) wasm
+all:
```

Comment:
The `all` target is no longer used. Should it just be removed?

Reply:
It was needed for gadgets with only the metadata file (like the one used in #3820), otherwise make tries to run the ebpf target. However, I fixed it by removing this and adding some extra logic to avoid invoking make if there's nothing to build.